### PR TITLE
Added lambda expressions

### PR DESCRIPTION
**Pull request by RoiEXLab**
Also IMO it makes no sense to check if the route is equal or not... the value is still the same...

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/653)

### CODE REVIEW

File: src/games/strategy/triplea/ui/MapPanel.java [hunk 1]
```diff
@@ -213,25 +213,11 @@ public void setRoute(final Route route) {
...
-      SwingUtilities.invokeLater(new Runnable() {
-        @Override
-        public void run() {
-          repaint();
-        }
-      });
-      return;
-    }
-    final RouteDescription newVal = new RouteDescription(route, start, end, cursorImage);
-    if (routeDescription != null && routeDescription.equals(newVal)) {
```
Comment (ron-murhammer) [inline src/games/strategy/triplea/ui/MapPanel.java]: Are we sure we should remove this check? It appears that if the route is the same as the existing one then we don't repaint it which kind of makes sense.

**Comment by RoiEXLab** [comment]
Yes you're probably right... I will readd this check

**Comment by RoiEXLab** [comment]
Finally got rid of all errors... That's why you don't code on a phone!

File: src/games/strategy/triplea/ui/MapPanel.java [hunk 2]
```diff
@@ -423,13 +398,7 @@ private boolean unitsChanged(final Tuple<Territory, List<Unit>> newUnits) {
...
     tileManager.updateTerritories(countries, m_data, uiContext.getMapData());
     smallMapImageManager.update(m_data, uiContext.getMapData());
-    SwingUtilities.invokeLater(new Runnable() {
-      @Override
-      public void run() {
-        smallView.repaint();
-        repaint();
-      }
-    });
+    SwingUtilities.invokeLater(() -> {smallView.repaint();repaint();});
```
Comment (ron-murhammer) [inline src/games/strategy/triplea/ui/MapPanel.java]: Please format this.

**Comment by ron-murhammer** [comment]
@RoiEXLab Just one format comment above to update then this should be good.

**Pull request merged**